src/Administration/Resources/app/administration/eslint-rules/deprecation-rules/no-deprecated-components.js

```diff
@@ -125,6 +125,10 @@ module.exports = {
                         'sw-progress-bar'
                     ];
 
+                    const preventDisableFixForFixedComponents = [
```

Comment:
This solution just prevents the fixer.

I wrote a more configurable alternative to this: https://github.com/shopware/shopware/pull/6713/files#diff-d1c30c721ffa63cdcdf621b274677fce97be28159b40edc352b865f3d3a1a5b0R207

This allows to show errors only for the activated components, etc. and fix them. Every non activated component gets completely ignored. The nice thing about this solution is that we can activate it for each component migration and be safe that nothing gets added afterwards